bundler/src/main.rs

```diff
@@ -75,13 +113,13 @@ fn process_package(path: &Path) -> anyhow::Result<ExtendedPackageInfo> {
 }
 
 /// Read and validate the package's manifest.
-fn parse_manifest(path: &Path) -> anyhow::Result<PackageManifest> {
+fn parse_manifest(path: &Path, namespace: &str) -> anyhow::Result<PackageManifest> {
     let src = fs::read_to_string(path.join("typst.toml"))?;
 
     let manifest: PackageManifest = toml::from_str(&src)?;
     let expected = format!(
-        "packages/preview/{}/{}",
-        manifest.package.name, manifest.package.version
+        "packages/{}/{}/{}",
+        namespace, manifest.package.name, manifest.package.version
```

Comment:
```suggestion
        "packages/{namespace}/{}/{}",
        manifest.package.name, manifest.package.version
```
might apply elsewhere